commands.js

```diff
@@ -370,6 +371,7 @@ var commands = exports.commands = {
 		Rooms.global.writeChatRoomData();
 	},
 
+	removealias:'removeroomalias',
```

Comment:
What I said about `alias` applies to `removealias` as well. We don't need this, either.